models/engine/db_storage.py

```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Method to retrieve one object"""
```

Comment:
Improve comments, a better description of how it retrieves the object. ex: retrieve based on class and id.